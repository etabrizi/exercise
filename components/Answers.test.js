import * as React from 'react';
import Answer from './Answers';
import renderer from 'react-test-renderer';

const Props = {
    content : "this is an answer",
    getValue: () => {},
    disabled: true,
    flag: true,
    correctAnswer : [{}],
    key: "this is an answer"
  }
  
describe('Answer component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Answer {...Props}/>)
            .toJSON();
        expect(tree).toMatchSnapshot()
    });
});   