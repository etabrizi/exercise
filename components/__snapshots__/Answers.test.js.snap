// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Answer component renders correctly 1`] = `
<li
  className="jsx-500512577"
>
  <input
    className="jsx-500512577"
    disabled={true}
    id="this is an answer"
    name="group"
    onChange={[Function]}
    type="radio"
    value="this is an answer"
  />
  <label
    className="jsx-500512577 good"
    htmlFor="this is an answer"
  >
    this is an answer
  </label>
</li>
`;
